src/View/Helper/TimeHelper.php

```diff
@@ -33,6 +34,121 @@ class TimeHelper extends Helper
     use StringTemplateTrait;
 
     /**
+     * The default locale to be used for displaying formatted date strings.
+     *
+     * @var string
+     * @deprecated Use self::setDefaultLocale() and self::getDefaultLocale() instead.
+     */
+    protected $_defaultLocale = null;
+
+    /**
+     * The default format to be used for displaying formatted date strings.
+     *
+     * @var string|null
+     */
+    protected $_defaultFormat = null;
+
+    /**
+     * The \DateTimeZone default output timezone.
+     * See http://php.net/manual/en/timezones.php.
+     *
+     * @var \DateTimeZone|null
+     */
+    protected $_defaultOutputTimezone = null;
+
+    /**
+     * Construct the TimeHelper
+     *
+     * @param \Cake\View\View $View The View this helper is being attached to
+     * @param array $config Configuration settings for the helper
+     */
+    public function __construct(View $View, array $config = [])
+    {
+        parent::__construct($View, $config);
+        if (isset($config['defaultLocale'])) {
+            $this->_defaultLocale = $config['defaultLocale'];
```

Comment:
Why do we need these properties instead of directly using the config values?  Also I don't think having the getter/setter methods are necessary in the helpers. One can just use `$helper->config('defaultOutputTimezone')` instead of `$helper->getDefaultOutputTimezone()`.